src/main/java/net/neoforged/neoforge/client/event/RenderLevelStageEvent.java

```diff
@@ -44,6 +46,7 @@ public class RenderLevelStageEvent extends Event {
     private final DeltaTracker partialTick;
     private final Camera camera;
     private final Frustum frustum;
+    private final Iterable<? extends IRenderableSection> renderableSections;
 
     public RenderLevelStageEvent(Stage stage, LevelRenderer levelRenderer, @Nullable PoseStack poseStack, Matrix4f modelViewMatrix, Matrix4f projectionMatrix, int renderTick, DeltaTracker partialTick, Camera camera, Frustum frustum) {
```

Comment:
It might make sense to provide a second constructor that does not pull in the sections from the level render, but takes it as an arg directly. That should make this api easier to use for other rendering mods

Reply:
I agree in theory, but in practice, rendering mods would end up needing to support `LevelRenderer#getVisibleSections` anyway, because there will inevitably be a content mod that uses it outside this event despite it being marked as internal.

Reply:
I'll still add it though, not hard. I just won't be needing it. :stuck_out_tongue: